### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by donizer**

### CODE REVIEW

File: src/arrayMethodSort.js [hunk 1]
```diff
@@ -4,8 +4,38 @@
...
+        if (String(a) < String(b)) {
+          return -1;
+        }
+
+        if (String(a) > String(b)) {
+          return 1;
+        }
+
+        return 0;
+      };
```
Comment (anastasiia-tilikina) [inline src/arrayMethodSort.js:25]: let's move your default fn outside your method and assign it as the default value for the parameter. In this case you can avoid checking type.

File: src/arrayMethodSort.js [hunk 2]
```diff
@@ -4,8 +4,38 @@
...
+    }
+
+    const length = this.length;
+
+    for (let i = 0; i < length - 1; i++) {
+      for (let j = 0; j < length - 1 - i; j++) {
+        if (compareFn(this[j], this[j + 1]) > 0) {
+          [this[j], this[j + 1]] = [this[j + 1], this[j]];
```
Comment (anastasiia-tilikina) [inline src/arrayMethodSort.js:33]: great!

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
well done! let's jus add some fixes

**Review by ivbutsykin — APPROVED** [review]
Good job!

**Pull request closed without merging**